packages/components/src/button/style.scss

```diff
@@ -223,6 +223,11 @@
 			color: color(theme(outlines) shade(25%));
 		}
 	}
+
+	// Fixes a Safari+VoiceOver bug, where the screen reader text is announced not respecting the source order.
```

Comment:
Could we include a reference link as part of the inline comment, perhaps to the Trac ticket or some of the debugging from the related HTML5 Boilerplate discussion, as without it, it'd be more difficult to revisit as part of future maintenance should the bug become resolved in Safari, or a better alternative proposed.

Reply:
Done.